tests/project-type/workspace-instance.spec.ts

```diff
@@ -1,24 +1,20 @@
-import { WorkspaceImpl } from "@sap/project-api";
 import { expect } from "chai";
-import { uniq } from "lodash";
-
+import * as _ from "lodash";
 import { mockVscode } from "../mockUtil";
 
 const testVscode = {
     workspace: {
-        workspaceFolders: () => []
+        workspaceFolders: () => [],
+        onDidChangeWorkspaceFolders: () => ""
     }
 };
 
 mockVscode(testVscode, "src/project-type/workspace-instance.ts");
 import { getBasWorkspaceAPI } from "../../src/project-type/workspace-instance";
 
-describe.skip("getBasWorkspaceAPI", () => {
-    it("multiple bas workspace instances should be equal", () => {
-        const basWsInstances = [];
-        for (let i = 0; i < 5; i++) {
-            basWsInstances.push(getBasWorkspaceAPI(WorkspaceImpl));
-        }
-        expect(uniq(basWsInstances)).to.have.lengthOf(1);
+describe("getBasWorkspaceAPI", () => {
+    it("multiple bas workspace instances should be identical", () => {
```

Comment:
I am thinking about this again and I am not sure:
- Do we really need this test?
- Does it matter if the `getBasWorkspaceAPI` returns the same instance?
- What is important is that for all our consumers via `vscode.extensions.getExtension("bas-toolkit").exports.workspaceApi` gets the same instance.

Do you remember why we did this test? I think I suggested it for some reason but it does not make sense for me now.

Reply:
I think that vscode already helps us to get same instance of basTollkitAPI.
I have added date property to basTollkitAPI in the createBasToolkitAPI method (Date.now())
In the sample example I have made 2 separate calls to app-studio-toolkit extension.
The dates property were equal.